hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceImpl.java

```diff
@@ -132,9 +132,10 @@ protected HazelcastInstanceImpl(String name, Config config, NodeContext nodeCont
 
         try {
             logger = node.getLogger(getClass().getName());
-            lifecycleService.fireLifecycleEvent(STARTING);
```

Comment:
When `node.start()` returns, node becomes started and joined to the cluster. To me `STARTING ` means the state before join, `STARTED` means node is ready to operate. 

Not sure but in this case, I don't see any significant difference between `STARTING` and `STARTED`.